quiz.js

```diff
@@ -0,0 +1,33 @@
+
+var growButton = document.getElementById("grow"); // variable representing button in DOM
+var treeLimbs = {}; // empty object to hold user input
+var space = " ";
+
+growButton.addEventListener("click", tree);
+
+function tree(treeLimbs) {
```

Comment:
While this seems to meet the requirements, what we want to see is the object passed in complete with it's key value pairs and not just an empty object.